common/v2/features/Dashboard/components/AccountDropdown.tsx

```diff
@@ -1,136 +1,124 @@
 import React, { Component } from 'react';
 import onClickOutside from 'react-onclickoutside';
+import styled from 'styled-components';
 import { Button, Identicon } from '@mycrypto/ui';
 
+import { Checkbox } from 'v2/components';
+import { ExtendedAccount } from 'v2/services';
 import './AccountDropdown.scss';
 
-interface AccountDropdownEntry {
-  uuid: string;
-  name: string;
-  address: string;
-  visible: boolean;
+interface DropdownProps {
+  accounts: ExtendedAccount[];
+  selected: string[];
+  onSubmit(selected: string[]): void;
 }
 
-interface Props {
-  accounts: AccountDropdownEntry[];
-  visibleCount: number;
-  allVisible: boolean;
-  onSelectAll(): void;
-  onSelect(uuid: string): void;
+interface State {
+  accounts: ExtendedAccount[];
+  selected: string[];
+  open: boolean;
 }
 
-export class AccountDropdown extends Component<Props, { open: boolean }> {
-  public state = {
-    open: false
+const Divider = styled('div')`
+  border-bottom: 1px solid #e5ecf3;
+  margin-bottom: 15px;
+`;
+
+const SButton = styled(Button)`
+  height: 40px;
+  width: 100%;
+  padding: 9px 0;
+  font-size: 18px;
+  line-height: 18px;
+  font-weight: bold;
+`;
```

Comment:
We should be able to use UI's Button instead of recreating a new one.

Reply:
The default styling of the UI button didn't correspond to the [zeplin designs] (https://app.zeplin.io/project/5b0334f5e91e8c481645ad56/screen/5be46a7c0f5ad13f1d1bc8ad), add looking at the [source](https://github.com/MyCryptoHQ/ui/blob/master/src/atoms/Button/Button.tsx) I didn't see any props to modify the height and the padding (`large={false}` has no effect).

So I choose to override the styles.
Please let me know if there is a better way!

Reply:
@nickmccurdy Would it be best to just override the UI Button directly here?

Reply:
I think so. You should probably start with ```styled(Button)`````, then you can start overriding with custom styles.

Reply:
@Sonicrida @nickmccurdy Yes this is how it's implemented. 
It's based of UI and uses `styled(Button)`.